tests/trainers/test_detection.py

```diff
@@ -17,7 +17,17 @@ class TestObjectDetectionTask:
     @pytest.mark.parametrize(
         "name,classname", [("nasa_marine_debris", NASAMarineDebrisDataModule)]
     )
-    def test_trainer(self, name: str, classname: Type[LightningDataModule]) -> None:
+    @pytest.mark.parametrize(
```

Comment:
I didn't even realize you could stack decorators like this to get the full cross product, this is way easier than what I was doing!